Schemas/documents.xsd

```diff
@@ -5,7 +5,13 @@
     <xs:element name="Documents">
         <xs:complexType>
             <xs:sequence>
-                <xs:element name="Documents" type="Document" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element name="Documents" minOccurs="0">
```

Comment:
This should allow only one occurence. If none are present the whole documents file should be omitted

Reply:
Fixed it exactly when you pointed it out. Thanks